docs/data-create.rst

```diff
@@ -1,73 +1,80 @@
+
+2. Download the dataset creation Docker container
+====================================================================
+
+Download the HistomicsML dataset creation Docker container
 
 .. code-block:: bash
 
   $ docker pull cancerdatascience/hml_dataset_gpu:1.0
 
-3. Create boundaries and centroids of superpixels.
+Use ``SuperpixelSegmentation.py`` to generate superpixel boundaries and centroids
 
 .. code-block:: bash
 
   $ docker run -it --rm --name createboundary -v "$PWD"/boundary:/boundary -v "$PWD"/centroid:/centroid cancerdatascience/hml_dataset_gpu:1.0 python scripts/SuperpixelSegmentation.py --superpixelSize 64 --patchSize 128
 
-.. note:: Adjustable parameters of ``SuperpixelSegmentation.py`` and ``FeatureExtraction.py`` are following::
+.. note:: 
+  Parameters of the superpixel segmentation script ``SuperpixelSegmentation.py`` can be adjusted to change the size, shape, and thresholding of superpixels to discard background regions
+  
+  -superpixelSize        An approximate edge length of each superpixel. Range is [8, 256]. Default 64.
+  
+  -patchSize             Patch size of each superpixel. Range is [8, 512]. Default 128.
 
---superpixelSize        An approximate edge length of each superpixel.
-                        Range is [8, 256]. Default 64.
---patchSize             Patch size of each superpixel. Range is [8, 512]. Default 128.
+3. Extract features
+====================================================================
 
-4. Create features of superpixels.
+On a CPU system
 
 .. code-block:: bash
 
-  # use the command line below if using CPU.
   $ docker run -it --rm --name extractfeatures -v "$PWD"/feature:/feature -v "$PWD"/centroid:/centroid cancerdatascience/hml_dataset_gpu:1.0 python scripts/FeatureExtraction.py --superpixelSize 64 --patchSize 128
-  # use the command line below if using GPU. Current verion supports CUDA 9.0, Linux x86_64 Driver Version >= 384.81
+
+On a GPU system (currently supporting CUDA 9.0, Linux x86_64 Driver Version >= 384.81):
+
+.. code-block:: bash
+
   $ docker run --runtime=nvidia -it --rm --name extractfeatures -v "$PWD"/feature:/feature -v "$PWD"/centroid:/centroid cancerdatascience/hml_dataset_gpu:1.0 python scripts/FeatureExtraction.py --superpixelSize 64 --patchSize 128
 
-5. Create HistomicsML dataset.
+.. note:: 
+  The patch size utilized by ``FeatureExtraction.py`` can be adjusted:
+
+4. Create HistomicsML dataset and check outputs
+====================================================================
+
+Transform the superpixel segmentation and feature information into the final .h5 format for ingestion
 
 .. code-block:: bash
 
   $ docker run -it --rm --name createdataset -v "$PWD"/dataset:/dataset -v "$PWD"/feature:/feature cancerdatascience/hml_dataset_gpu:1.0 python scripts/CreateDataSet.py
 
-6. Outputs.
+Check the generated outputs
```

Comment:
@slee172 I didn't understand the note about the default .h5 file name "BRCA-spfeatures-2.h5". How is this controlled with arguments to createdataset?

Reply:
@cooperlab I just updated this data creation. We are no longer using "CreateDataSet.py". And, we have an argument "outputDataSetName" that indicates a dataset name on "FeatureExtraction.py" (the default dataset name is "BRCA-spfeatures-2.h5"). So, for example, "FeatureExtraction.py --outputDataSetName BRCA-spfeatures-2.h5" will create a dataset named "BRCA-spfeatures-2.h5".

Reply:
I updated this documentation with more appropriate file names.